src/daemon/command_parser_executor.cpp

```diff
@@ -203,7 +203,21 @@ bool t_command_parser_executor::prepare_registration()
 
 bool t_command_parser_executor::print_sn(const std::vector<std::string>& args)
 {
-  bool result = m_executor.print_sn(args);
+  bool include_json = false;
```

Comment:
print_sn command needs to be updated so the help text says that you can pass the +json flag to it like print_tx does.